scripts/st2bootstrap-deb.sh

```diff
@@ -474,8 +474,9 @@ install_st2mistral() {
   # Setup Mistral DB tables, etc.
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
 
-  # Register mistral actions
-  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+  # Register mistral actions. Init of git is added here to temporarily fix
+  # a bug in one of the OpenStack dependency related to pbr version checking.
+  git init && /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate && rm -rf .git
```

Comment:
@m4dcoder to reduce probability of this in a future and get rid of git dependency, 
would you consider upstream change in `mistral-db-manage` to make registration of openstack actions optional?